src/sentry/models/project.py

```diff
@@ -280,7 +280,12 @@ def is_user_subscribed_to_mail_alerts(self, user):
             is_enabled = bool(is_enabled)
         return is_enabled
 
-    def transfer_to(self, team):
+    def transfer_to(self, team=None, organization=None):
+
+        # TODO(jess): remove this when new-teams is live for everyone
+        # only support passing team or org not both
+        assert (team and not organization) or (organization and not team)
```

Comment:
Isn't this the same as `assert not (team and organization)` ?

Reply:
damn that's way better.